packages/near-wallet-selector/src/wallets/injected/SenderWallet.ts

```diff
@@ -82,7 +83,7 @@ function setupSenderWallet(): WalletModule<InjectedWallet> {
             await this.signOut();
             await this.signIn();
           } catch (e) {
-            logger.log(`Failed to change account ${e.message}`);
+            logger.log(`Failed to change account ${(e as unknown as Error).message}`);
```

Comment:
Was this an auto fix? 😆 I'll let some of this slide as we're aware the ESLint configuration needs to be reintegrated maintaining compatibility with NX